tests/daily_scraper/test_daily_scraper.py

```diff
@@ -256,6 +256,7 @@
     # -p mac64 --branch=mozilla-central
     {'args': {'application': 'b2g',
               'platform': 'win32',
+              'extension': 'zip',
```

Comment:
We shouldn't have to change those tests, given that we fallback to the default extension in the scraper.